react-native-gutenberg-bridge/index.js

```diff
@@ -136,4 +136,12 @@ export function logUserEvent( event, properties ) {
 	return RNReactNativeGutenbergBridge.logUserEvent( event, properties );
 }
 
+export function getStarterPageTemplatesTooltipShown( callback ) {
```

Comment:
As more of a question more than a suggestion: 

I see [one other spot](https://github.com/wordpress-mobile/gutenberg-mobile/blob/2ee4845b56201610b58a34065a58be160a4767b4/react-native-gutenberg-bridge/index.js#L116) where `get` is used to request information but mostly it seems like we're using the keyword `request` on most calls. 

Is there a pattern where the Gutenberg Bridge is using one over the other?

Reply:
Fwiw, I had a similar thought while reviewing, but forgot to mention it. Your comment reminded me :smile: . I generally like the semantics of "request" better than "get" when there is a callback involved, instead of the function directly returning the value. Functions over the bridge are an interesting case though, because we _must_ use a callback interface to retrieve data from the parent app, even when the data is guaranteed to be available, and the callback _must_ be invoked. Following this convention strictly, I suppose is a mild forfeit of some expressibility, in a sense. It seems that perhaps there is a pattern emerging where "get" signifies an "almost-sort-of" locality of reference, but I'm not sure whether that is intentional.

Reply:
Thanks for the feedback! I agree `request` would make better sense and it'll consistent with the rest of the calls. I'll change it =)